recipes/ignition-cmake/all/test_package/CMakeLists.txt

```diff
@@ -4,9 +4,8 @@ project(test_package VERSION 1.0)
 include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
 conan_basic_setup()
 
-find_package(ignition-cmake2 REQUIRED CONFIG)
+find_package(ignition-cmake2 REQUIRED)
 
-include(IgnUtils)
-
-ign_setup_packages()
+include(IgnCMake)
```

Comment:
Why does need to replace these include and function?

Reply:
The test package shows how to import all IgnCMake functionalities and not just the functionalities in IgnUtils. (See the file in original github repo [here](https://github.com/ignitionrobotics/ign-cmake/blob/ign-cmake2/cmake/IgnCMake.cmake#L25)) The recipe that existed originally imported IgnUtils. This worked fine since the config.cmake files provided by library authors was directly used in the generated package instead of relying on the information generated by Conan

The correct way to do this IMO would be to generate information for path of  IgnCMake via patches (done [here](https://github.com/conan-io/conan-center-index/blob/eb967ab5c63f5882a22d904b339f437f20ec2ac1/recipes/ignition-cmake/all/patches/0002-ignition-cmake-2.10.0-fixes.patch#L8) in this MR) and use this information to get all the functionalities from ignition-utilities